src/Components/UI/Calculator.js

```diff
@@ -1,47 +1,110 @@
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prefer-stateless-function, react/destructuring-assignment, no-nested-ternary, max-len */
+// /* eslint-disable */
 import React from 'react';
+import calculate from '../Logic/calculate';
 import './Calculator.css';
 
 class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0,
+      next: 0,
+      operation: null,
+    };
+  }
+
+  addInput = (event) => {
+    const clickedNumber = event.target.innerText;
+    if (this.state.operation === null && this.state.next === null) {
+      if (this.state.total === 0 || this.state.total === '0') {
+        // eslint-disable-next-line react/no-direct-mutation-state
+        this.state.total = '';
+      }
+      // eslint-disable-next-line react/no-access-state-in-setstate
+      const temp = this.state.total.toString() + clickedNumber.toString();
+      this.setState({ total: temp });
+    } else if (this.state.operation !== null && this.state.total !== null) {
+      if (this.state.next === null) {
+        this.setState({ next: clickedNumber.toString() });
+      } else {
+        // eslint-disable-next-line no-unused-expressions
+        // eslint-disable-next-line react/no-access-state-in-setstate
+        const temp = this.state.next.toString() + clickedNumber.toString();
+        this.setState({ next: temp });
+      }
+    }
```

Comment:
- [ ] Kindly remove the disable linters as linters help us write clean industry acceptable code.